fusedrug/data/tokenizer/ops/modular_tokenizer_ops.py

```diff
@@ -266,7 +266,7 @@ def __call__(
         ):  # note, encoded.overflowing may have multiple items, and each item can contain multiple items
             print(
                 f"Warning: FastModularTokenizer (pid={os.getpid()}) had to truncate sequence: [{overflow_info}]  \
-                    for tokenizer: {self._tokenizer_path} for sample_id {get_sample_id(sample_dict)}"
+                    for tokenizer: {self._tokenizer_path} for sample_dict={sample_dict}"
```

Comment:
This will probably too noisy. Do you have any other suggestion? (In case `sample_dict` doesn't have `sample_id`)

Reply:
Another option is to make sure (and enforce) keeping `data.sample_id` key.
For example, in my diffusion task I don't keep it in the data processing.